Tests/TensorFlowTests/OperatorTests/MathTests.swift

```diff
@@ -58,6 +58,84 @@ final class MathOperatorTests: XCTestCase {
         testElementaryFunction(name: "root",
                                { x in root(x, 3) }, { x in Float.root(x, 3) })
     }
+    
+    func testAbs() {
+        let x = Tensor<Float>([-1.0])
+        let y = abs(x)
+        assertEqual(y, Tensor<Float>([1.0]))
+    }
+    
+    func testSquaredDifference() {
+        let x = Tensor<Float>([-5.8])
+        let y = Tensor<Float>([-5.7])
+        let z = squaredDifference(x, y)
+        let zaprox = 132.25
+        assertEqual(z, zaprox, accuracy: 0.0001)
+    }
+    
+    func testZeros() {
+        let x = Tensor<Float>(zeros:[1])
+        let x1 = Tensor<Float>([0.0])
+        assertEqual(x, x1)
+    }
+    
+    func testLogSoftmax() {
+        let x = Tensor<Float>([32.0, 34.0, 35.0])
+        let y = logSoftmax(x)
+        let y1 = Tensor<Float>([-3.34901, -1.34901, -0.34901])
+        assertEqual(y, y1, accuracy: 0.0001)
+    }
+    
+    func testLogCoshLoss() {
+        let predicted = Tensor<Float>([32.0, 34.0, 36.0])
+        let expected = Tensor<Float>([32.0, 36.0, 37.0])
+        let val = logCoshLoss(predicted:predicted , expected:expected)
+        let valExpected = 0.58626
+        assertEqual(val, valExpected, accuracy: 0.0001)
+    }
+    
+    func testMax() {
+        let x = Tensor<Float>([58.0])
+        let y = Tensor<Float>([57.0])
+        let z = max(x, y)
+        let zExpected = 58.0
```

Comment:
```suggestion
        let expectedZ= 58.0
```
And the subsequent assertion should be updated as well. Same for all other tests.